js-exercises/last-index-of/lastIndexOf.js

```diff
@@ -0,0 +1,14 @@
+
+function lastIndexOf(...args) {
+  const item = args[0];
```

Comment:
You can use itemToSearch const name instead of item as it helps to understand the code easily.